src/daos-deserialize/daos-deserialize.c

```diff
@@ -166,6 +167,7 @@ int main(int argc, char** argv)
     int option_index = 0;
     static struct option long_options[] = {
         {"pool"                 , required_argument , 0, 'p'},
+        {"cont_label"           , required_argument , 0, 'l'},
```

Comment:
I think to be more consistent with other tools/options
```suggestion
        {"cont-label"           , required_argument , 0, 'l'},
```

Also, should we just make this more generic and call it `cont` so the user can pass either a label or uuid?

Reply:
@daltonbohning I am not sure on this because I don't really see a reason to want to generate a uuid then pass it into the program when dcp will just do it for you in this case. I don't have very strong opinions about it I guess allowing either is more consistent.

Reply:
@dsikich Hmm. I'm not sure if a user would necessarily pass in a generated UUID, but yeah, I was more so thinking about consistency with other tools. But then again, even `daos cont create` has a `--label` option... so I'm not sure. Maybe we can leave it for now and ask the larger group at the next meeting.